apps/src/p5lab/spritelab/poembot/commands/foregroundEffects.js

```diff
@@ -265,8 +267,39 @@ export const commands = {
         break;
       case 'birds':
         break;
-      case 'twinkling':
+      case 'twinkling': {
+        let stars = [];
+        for (let i = 0; i < 75; i++) {
+          stars.push({
+            color: utils.randomColorFromPalette('twinkling'),
+            x: utils.randomInt(0, 400),
+            y: utils.randomInt(0, 400),
+            alpha: utils.randomInt(1, 100),
+            delta: this.p5.random([-6, -5, -4, -3, 3, 4, 5, 6])
```

Comment:
i don't think i understand what `delta` and the values in this array mean here -- can you add a comment?